package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Feb 27 21:19:45 UTC 2019 - Dominique Leuenberger <dimstar@opensuse.org>
+
+- Default to wicked on generic desktop: do the same as we did
+  before yast learned to auto-pick NetworkManager if the
+  control.xml happened to request it).
```

Comment:
Hehe, well, YaST already know about it, but the proposal was not part of openSUSE until #170 

:) 

There is also a close parenthesis that was never opened :P and please also mention the bug number (bsc#1127228)

Reply:
Wasn't #170 'just' the addition to SHOW the proposal? I think the actual change causing the fallout was yast2-network going the extra mile to be smarter about the choice (because, with it added to the overview, the users were easier to change the selection)

as for the closing parenthesis and bug ref: will fix

Reply:
Yep, it can be definitely be described in that way.